client/profile-wizard/steps/store-details/style.scss

```diff
@@ -7,6 +7,10 @@
 	.components-popover .components-popover__content {
 		min-width: 360px;
 	}
+
+	.woocommerce-store-address-fields .woocommerce-select-control label.components-base-control__label {
```

Comment:
Would this fix be better suited in the select-control styles so that it affects all `SelectControl`s?

Reply:
Yeah, it would; I'm just being shy. I'll make that change.

Reply:
Okay, learning about how the wca packages work was a fun little diversion for me today, but what I actually wound up doing rather than adding a rule to the SelectControl is just remove the specific rule that I needed to override in the first place.